### DISCUSSION TITLE
Consider breaking out Podcast App for US

### DISCUSSION THREAD

**Discussion by tomn68** [skymethod/livewire-web#3] [Ideas]
It would be helpful to know the Apple vs Spotify breakout for just the US vs Europe+.  Reason being if you have a US only podcast, it would be good to know how Spotify is comparing to Apple since I suspect there's a reasonable difference if you back out Europe/Asia.

FWIW, Libsyn says they're seeing 3.5 to 1, Apple to Spotify (according to their podcast episode The Feed last week) which is very different than Buzzsprouts stats unless I'm misinterpreting.

**Comment by johnspurlock-skymethod** [comment]
> It would be helpful to know the Apple vs Spotify breakout for just the US vs Europe+. 

Which post is this referring to?

I keep track of Libsyn's stats from The Feed here: https://livewire.io/libsyn-stats-visualized/
(it's not 3.5 to 1, it's 2 to 1 and dropping)

and Buzzsprout's stats here: https://livewire.io/buzzsprout-stats-visualized/
(Spotify is now ahead here)

Really it depends on where the listeners to their large shows are, but in general Spotify has been taking share everywhere
